pages/programs/baton_roots.html

```diff
@@ -1 +1,30 @@
+<!DOCTYPE html>
+<html>
 
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width">
+	<title>repl.it</title>
+	<link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+	<script src="script.js">
+
+	</script>
+	<div class="box-content">
```

Comment:
Can we update this to be `youth-managed-farm` since we follow that pattern for the children.